### PR TITLE
Specify "recent JDK build"

### PR DESCRIPTION
**Pull request by lrytz**
https://issues.scala-lang.org/browse/SI-9828 / https://issues.scala-lang.org/browse/SI-10041

### CODE REVIEW

**Comment by lrytz** [comment]
@SethTisue, wdyt?

File: hand-written.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ We hope to address the following in a later 2.12.x release:
 
 ### Java 8 runtime
 
-Install a recent build of the Java 8 Platform, such as [OpenJDK](http://openjdk.java.net/install/) or [Oracle Java](http://www.oracle.com/technetwork/java/javase/downloads/index.html). Any Java 8 compliant runtime will do.
+Install a recent build of the Java 8 Platform, such as [OpenJDK](http://openjdk.java.net/install/) or [Oracle Java](http://www.oracle.com/technetwork/java/javase/downloads/index.html) (versions before 8u112 have a known issue that [affects Scala](https://issues.scala-lang.org/browse/SI-9828)). Any Java 8 compliant runtime will do.
```
Comment (retronym) [inline hand-written.md]: Maybe [8u102](http://www.oracle.com/technetwork/java/javase/2col/8u102-bugfixes-3021768.html) is enough: https://bugs.openjdk.java.net/browse/JDK-8149473

Comment (lrytz) [reply hand-written.md]: ok, changed that

**Comment by SethTisue** [comment]
LGTM. I will propagate the change to the [blog post](http://www.scala-lang.org/news/2.12.0) and [GitHub release](https://github.com/scala/scala/releases/tag/v2.12.0).

also tweeted: https://twitter.com/scala_lang/status/797162344413413376

**Comment by SethTisue** [comment]
rearranged slightly in b54c32f6e0387e63fc5aba2753c3aa6489a01876

**Comment by lrytz** [comment]
thank you!

**Pull request merged**